lib/plugins/aws/deploy-function.js

```diff
@@ -208,6 +208,112 @@ class AwsDeployFunction {
     await callWithRetry();
   }
 
+  static serverlessConsoleLayerConfig(remoteFunctionConfiguration) {
```

Comment:
Let's not introduce static functions. I know it was created for test purposes, but we should not attach to implementation details in tests (focus only outcome testing via `runServerless` runs)